tests-legacy/Endpoints/AbstractEndpointAdminTest.php

```diff
@@ -44,6 +54,15 @@ protected function setUp()
             define('_PS_TAB_MODULE_LIST_URL_', '');
         }
         Context::getContext()->employee = new Employee(1);
+
+        $this->tokenStorage = SymfonyContainer::getInstance()->get('security.token_storage');
+    }
+
+    protected function tearDown()
+    {
+        parent::tearDown();
+
+        SymfonyContainer::getInstance()->set('security.token_storage', $this->tokenStorage);
```

Comment:
I don't understand the purpose of this `tearDown` method I can understand you need the token storage in tests an need to get it in the setUp method
But whats the point of overriding the service in the container during tearDown (especially if you set the same value)?

Reply:
explained in comments below